lib/src/scrollbar95.dart

```diff
@@ -0,0 +1,383 @@
+import 'dart:async';
+import 'dart:math';
+import 'dart:ui';
+
+import 'package:flutter/material.dart';
+import 'package:flutter95/flutter95.dart';
+
+class Scrollbar95 extends StatefulWidget {
+  final Widget child;
+  final double scrollAmount;
+  final double minThumbHeight;
+  final double scrollbarThickness;
```

Comment:
Small improvement, but you could add the same docs you added to the readme to the class directly as `///` comments.

Reply:
Sure